### DISCUSSION TITLE
Block tag with itemstack

### DISCUSSION THREAD

**Discussion by DevBlokJochem** [Minestom/Minestom#1117] [Help]
Hello, I want to set itemstack data on a block using tags. This is the code that im using right now: 
```
val newBlock = inv.getBlock()
  .withTag(Tag.ItemStack("inventoryData").list(), Arrays.stream(itemStacks).toList()) 
inv.setBlock(newBlock()
println(inv.getBlock().getTag(Tag.ItemStack("inventoryData").list()).toString)
```
The problem is, that I get an error in the console that tells me that the Tag is null. 
What should happen is that the console tells me some itemstack data so Im sure that everything is saved.

Im working on a chest save inventory thing. So I have created a BlockInventory class that contains functions like .setBlock and .getBlock. So I am able to get the block from the inventory.

How can I change my code to something that will work?

**Comment by TheMode** [comment]
Seems like the issue was Minestom side. Please try the commit f5dc0481850d7bb579fefbe318da14101faf1f53
